src/compiler/utilities.ts

```diff
@@ -10116,11 +10116,11 @@ export function skipTypeChecking(sourceFile: SourceFile, options: CompilerOption
         options.skipDefaultLibCheck && sourceFile.hasNoDefaultLib) ||
         options.noCheck ||
         host.isSourceOfProjectReferenceRedirect(sourceFile.fileName) ||
-        !canIncludeBindAndCheckDiagnsotics(sourceFile, options);
+        !canIncludeBindAndCheckDiagnostics(sourceFile, options);
 }
 
 /** @internal */
-export function canIncludeBindAndCheckDiagnsotics(sourceFile: SourceFile, options: CompilerOptions) {
+export function canIncludeBindAndCheckDiagnostics(sourceFile: SourceFile, options: CompilerOptions) {

```

Comment:
Nit, but I would suggest sending this typo fix to main just to get it out of this PR's diff.

Reply:
#58840

Reply:
Sorry about the typo and extra work you had to do @gabritto